### ISSUE TITLE
[Accessibility] The right-most tab needs a label, and extraneous items should be removed.

### ISSUE THREAD

**Issue by MarcoZehe** [DagAgren/toot#127]
The tab at the bottom has a few problems after the Notifications tab.

First problem: The Local Timeline tab is missing an AccessibilityLabel.

Second problem: When swiping right from Notifications, VoiceOver lands on items that say "Greater than", as in ">", and other funny stuff. These should be found and set to be IsAccessibilityElement: No.
Only the remaining Local Timeline tab should remain in the order.

**Comment by DagAgren** [comment]
Thanks! Feel free to post any other accessibility issues, since I am still unfamiliar with that and can use any and all help!
